internal/service/mailing/mailing.go

```diff
@@ -0,0 +1,100 @@
+package service
+
+import (
+	"fmt"
+	dto "optitech/internal/dto/mailing"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/joho/godotenv"
+	"gopkg.in/gomail.v2"
+)
+
+func CreateMailingService(send dto.MailingReq) error {
```

Comment:
```suggestion
func SendPassword(send dto.MailingReq) error {
```
Ya sabemos que es un servicio de mail, pero específicamente este que hace?, en este caso envía password.

Reply:
Cambiado: https://github.com/OwlByTech/optitech-api/pull/3/commits/8f279e7ce2b4af6dbada59de0c05143176aa1ff1